src/shared/organisms/DataPanel/DataPanel.spec.ts

```diff
@@ -1,14 +1,13 @@
-import { vi } from 'vitest';
+import { vi as jest } from 'vitest';
```

Comment:
why this change? Maybe it's better to rename the occurences of the word `jest` underneath to `vitest`.

Reply:
you are right about it i fixed it, in the next PR